docs/tutorial/part-seven/index.md

```diff
@@ -317,39 +325,56 @@ The last step is to link to your new pages from the index page.
 Return to `src/pages/index.js` and let's query for your markdown slugs and create
 links.
 
-```jsx{3,18-19,29,47-49}
+```jsx{3,23-29,45,64-66}
 import React from "react"
-import g from "glamorous"
+import { css } from "react-emotion"
 import { Link } from "gatsby"
-
 import { rhythm } from "../utils/typography"
+import Layout from "../components/layout"
 
 export default ({ data }) => {
+  console.log(data)
```

Comment:
Is this intentional?

Reply:
Good catch. This was a line from part-six that I missed to remove. Fixed.